core/roboconf-agent/src/main/java/net/roboconf/agent/internal/Agent.java

```diff
@@ -82,7 +81,7 @@
 	 */
 	public Agent() {
 		this.logger = Logger.getLogger( getClass().getName());
-		this.messagingFactoryType = MessagingConstants.FACTORY_RABBIT_MQ;
+		this.messagingType = MessagingConstants.FACTORY_RABBIT_MQ;
```

Comment:
There should not be any default value for the messaging type.
Indeed, this value only makes sense if the implementation bundle is also deployed.